src/web/components/Core/Dialog.scss

```diff
@@ -0,0 +1,68 @@
+.dialog-overlay {
+  background-color: rgba(255, 255, 255, 0.8);
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.dialog-content {
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0px 4px 14px rgba(0, 0, 0, 0.25);
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 90vw;
+  max-width: 450px;
+  max-height: 85vh;
+  padding: 47px;
+  animation: contentShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+  height: auto;
+
+  .icon-button {
+    position: absolute;
+    top: 47px;
+    right: 47px;
+    background: none;
+  }
+
+  h2 {
```

Comment:
Radix usually recommends using a className and targeting that for styling - but if you've seen something about the dialog maybe this is fine?